package.json

```diff
@@ -15,11 +15,9 @@
     "lint": "eslint lib/**/*.js test/**/*.js",
     "test": "npm run test:server && npm run test:features",
     "test:server": "mocha \"test/**/*.test.js\"",
```

Comment:
I wouldn't call this a server `server`, there's no server in Gavel. It's more like `test:unit` or `test:mocha`.

Reply:
Agree. Renamed in e61a532.